src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceEntry.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+/**
+ * <summary>Displays the high-level information for a Sequence</summary>
+ */
+public class RGSequenceEntry : MonoBehaviour, IPointerEnterHandler, IPointerExitHandler
+{
+    public string name;
+
+    public string description;
+
+    public DateTime lastModified;
+
+    /**
+     * UI component fields
+     */
+    [SerializeField]
+    public TMP_Text nameComponent;
+
+    [SerializeField]
+    public TMP_Text descriptionComponent;
+
+    [SerializeField]
+    public TMP_Text lastModifiedComponent;
+
+    /**
+     * Hover mangement. Assign a UI component that will be shown by default,
+     * and another UI component that will hide by default (ie: be shown only when
+     * the Sequence is hovered)
+     */
+    [SerializeField]
+    public Component showByDefault;
+
+    [SerializeField]
+    public Component hideByDefault;
+
+    void Start()
+    {
+        if (nameComponent != null)
```

Comment:
Will things work if these fields are not set? If they will cause a crash we can consider telling the user things are missing right at this level, instead of having a downstream crash with a harder to debug error message.

Reply:
These `*Component` fields are set by us! In this file, the *Component fields have all been manually set in their parent prefab. It is possible that this assignment gets messed up by us when updating the prefab in the future, but in the worst case these fields will just be empty (eg: if the `nameComponent` isn't set, we wont' see the name on the Sequence Card)

Having said all of this, I'm a complete novice with Unity dev. If there's any improvements I can make at all, I'm game!